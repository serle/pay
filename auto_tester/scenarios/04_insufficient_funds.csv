type,client,tx,amount
deposit,1,1,50.0
withdrawal,1,2,100.0
withdrawal,1,3,25.0
