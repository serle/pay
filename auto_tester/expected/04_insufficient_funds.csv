client,available,held,total,locked
1,25.0000,0.0000,25.0000,false
